### PR TITLE
Clarify documentation for --cxx17ns

### PR DESCRIPTION
**Pull request by kimgr**
Issue #1654 righfully complained that the docs were opaque; somehow we forgot to mention it controls namespace declarations. Take the suggested doc string, and apply it to both --help output and manpage.

### CODE REVIEW

**Comment by kimgr** [comment]
@carlosgalvezp Thanks for the suggestion, does this look good to you?

File: include-what-you-use.1 [hunk 1]
```diff
@@ -63,7 +63,7 @@ default.
 .RE
 .TP
 .B \-\-cxx17ns
-Suggest the more concise syntax for nested namespaces introduced in C++17.
+Use C++17 nested namespaces when suggesting additions of forward declarations.
```
Comment (carlosgalvezp) [inline include-what-you-use.1:66]: Ultra-nit: Here "Use" starts with capital U whereas in the other file it's small u.

**Comment by carlosgalvezp** [comment]
Looks great, thank you!

File: include-what-you-use.1 (hunk 1 above)
Comment (kimgr) [reply include-what-you-use.1:66]: That's intentional, to vibe with the surrounding code

**Pull request merged**